libs/agent-sm/bootstrap/src/main/java/org/opensearch/javaagent/bootstrap/AccessController.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.javaagent.bootstrap;
```

Comment:
I don't know the right module for this code. The server has a dependency on this lib, but its marked as `compileOnly`. How are the other classes in this module (like AgentPolicy) available at runtime?

Reply:
I believe jars passed to the JVM via `-javaagent` are available on the classpath, so the compileOnly dependency is making the assumption this will be provided at runtime via a `-javaagent`.